README.md

```diff
@@ -194,6 +194,15 @@ api.MetricsListSubscriptions(&Cursor{}, "customerUUID")
 api.MetricsListActivities(&Cursor{}, "customerUUID")
 ```
 
+### CSV Upload
+
+Available CSV Upload methods:
+
+
+```go
+api.UploadCSVFile(filePath string, uploadRequest *CsvUploadRequest)
```

Comment:
You should probably also implement a method that will upload from a `io.Reader`. Not everyone wants to first dump the data on the disk. But there's a catch if you need to retry, you'd need to rewind.

@viktorchukhantsev how will you do it in the service?

Reply:
changed it to an `interface{}` that accepts a `string` considered as a file path or an `io.Reader`